### PR TITLE
Add additional item IDs

### PR DESCRIPTION
**Pull request by mike8699**
Hi there, I'm one of the devs working on the [phantom hourglass randomizer](https://github.com/phst-randomizer/ph-randomizer). I noticed you are enumerating the item IDs so I went ahead and added a bunch of missing ones that we've documented during the course of developing the randomizer. Hope this helps!

### CODE REVIEW

File: include/Item.hpp [hunk 1]
```diff
@@ -68,19 +76,65 @@ enum ItemId_ {
 
     ItemId_Hammer = 31,
     ItemId_Rope = 32,
+    ItemId_SquareCrystal = 33,
+    ItemId_SquareCrystal = 34,
+    ItemId_SquareCrystal = 35,
```
Comment (AetiasHax) [inline include/Item.hpp]: The triangle and circle crystals are missing here!

**Review by AetiasHax — CHANGES_REQUESTED** [review]
Thanks a ton! I got word of the randomizer recently, and from your PH docs, I noticed that the Player struct (0x021BA4DC) corresponds with the ItemManager class in this repo. So, I shamelessly copied over what was missing in ItemManager from your docs. Please feel free to copy it back. 😉

Anyway, there's a small error near the shape crystals; they're all named the same. Fix it when you have the time!

File: include/Item.hpp (hunk 1 above)
Comment (mike8699) [reply include/Item.hpp]: Whoops 😅

Comment (mike8699) [reply include/Item.hpp]: Fixed!

**Pull request merged**